### PR TITLE
New enemy

### PR DESCRIPTION
**Pull request by Lufrauwu**

### CODE REVIEW

File: Assets/Scripts/Enemies/ChaseAndReturnEnemy.cs [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+public class ChaseAndReturnEnemy : MonoBehaviour
+{
+    [SerializeField] private GameObject _player = default;
+    [SerializeField] private float _distance = default;
+    [SerializeField] private float _speedEnemy = default;
+    [SerializeField] private float _enemyHealth = default;
+    private Transform _playerPosition = default;
+    private Vector2 _currentPosition = default;
+    void Start()
```
Comment (MikeLordx) [inline Assets/Scripts/Enemies/ChaseAndReturnEnemy.cs:12]: espacio aqui

File: Assets/Scripts/Enemies/ChaseAndReturnEnemy.cs [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    public IEnumerator Freeze()
+    {
+        _speedEnemy = 0;
+       // _enemyAnimator.gameObject.GetComponent<Animator>().enabled = false;
+        gameObject.GetComponent<SpriteRenderer>().color = Color.cyan;
+        yield return new WaitForSeconds(3.0f);
+        _speedEnemy = 10;
+       // _enemyAnimator.gameObject.GetComponent<Animator>().enabled = true;
+        gameObject.GetComponent<SpriteRenderer>().color = Color.white;
+
```
Comment (MikeLordx) [inline Assets/Scripts/Enemies/ChaseAndReturnEnemy.cs:55]: delete this space

File: Assets/Scripts/PlayerScripts/HealLife.cs [hunk 3]
```diff
@@ -0,0 +1,32 @@
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+public class HealLife : MonoBehaviour
+{
+    private HealthController healthController = default;
+    private PlayerController _playerController = default;
+    private InputAction _inputHeal = default;
+    void Start()
```
Comment (MikeLordx) [inline Assets/Scripts/PlayerScripts/HealLife.cs:10]: add space here i will not review you if you ever do this again >:(

File: Assets/Scripts/PlayerScripts/HealLife.cs [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    {
+        _playerController = new PlayerController();
+        _playerController.Enable();
+        _inputHeal = _playerController.Land.Heal;
+        _inputHeal.Enable();
+        _inputHeal.started += _ => Healing();
+    }
+    private void OnDestroy()
```
Comment (MikeLordx) [inline Assets/Scripts/PlayerScripts/HealLife.cs:18]: aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaah seriously? I just told you

File: Assets/Scripts/PlayerScripts/HealLife.cs [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    {
+        _playerController.Disable();
+        _inputHeal.Disable();
+    }
+
+    void Update()
+    {
+        
+    }
+    private void Healing()
```
Comment (MikeLordx) [inline Assets/Scripts/PlayerScripts/HealLife.cs:24]: I cant do this all day you just owe me a big carl

File: Assets/Scripts/PlayerScripts/Portal.cs [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+public class Portal : MonoBehaviour
+{
+    [SerializeField] private GameObject _darkDimention = default;
+    [SerializeField] private GameObject _normalDimention = default;
+    [SerializeField] TMP_Text _Interacttext = default;
+    private bool _canInteract = default;
+    private PlayerController _playerController = default;
+    private InputAction _inputDimention = default;
+    
+
```
Comment (MikeLordx) [inline Assets/Scripts/PlayerScripts/Portal.cs:15]: more spaces? Delete the extra >:(

**Review by MikeLordx — CHANGES_REQUESTED** [review]

File: Assets/Scripts/PlayerScripts/HealLife.cs (hunk 3 above)
Comment (Lufrauwu) [reply Assets/Scripts/PlayerScripts/HealLife.cs:10]: ya viejo loco

File: Assets/Scripts/PlayerScripts/HealLife.cs (hunk 4 above)
Comment (Lufrauwu) [reply Assets/Scripts/PlayerScripts/HealLife.cs:18]: que ya est{a viejo loco aprenda a lerr

File: Assets/Scripts/PlayerScripts/HealLife.cs (hunk 5 above)
Comment (Lufrauwu) [reply Assets/Scripts/PlayerScripts/HealLife.cs:24]: silencio

File: Assets/Scripts/PlayerScripts/HealLife.cs (hunk 4 above)
Comment (MikeLordx) [reply Assets/Scripts/PlayerScripts/HealLife.cs]: reprobado

**Review by MikeLordx — APPROVED** [review]

**Pull request merged**